pyproject.toml

```diff
@@ -27,7 +27,7 @@ omit = ['*/tests/*']
 
 [tool.black]
 line-length = 88
-target-version = ['py36']
+target-version = ['py38']
```

Comment:
This should be the minimum python version we accept, no? Also, still out of scope. This is already touching so many things I'd prefer that there were no extraneous changes for ease of review.